.github/workflows/ci.yml

```diff
@@ -41,7 +41,7 @@ jobs:
     needs: build-dependencies
     strategy:
       matrix:
-        java: [ 11 ]
+        java: [ 11, 17 ]
```

Comment:
I don't think we should validate the format twice, one for 11 and another one for 17.